src/main/java/th/ac/kmitl/soa/group2/forms/HeaderForm.java

```diff
@@ -29,6 +30,7 @@
 
     @Override
     public HeaderModel toModel() {
+        HeaderNoteModel headerNoteModel = new HeaderNoteModel("หมายเหตุ", "เนื้อหา");
```

Comment:
need you help to check this line .

Reply:
I'd prefer this to variable assignment

``` java
return new HeaderModel(
    id,
    DocumentType.from(typeCode).get().typeCode,
    typeCode,
    issuedAt,
    Option.some("dummy purpose"),
    purposeCode,
    globalId,
    Time.now(),
    new headerNoteModel(
        "หมายเหตุ",
        "เนื้อหา"
    )
);
```

And I think we can generalize it to NoteModel as you said it can be reused in [3.4.2] again